openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -865,8 +865,10 @@ Here is another example JWT not only proving possession of a private key but als
 
 ```json
 {
+  "typ": "openid4vci-proof+jwt",
   "alg": "ES256",
-  "x5c": [<key certificate + certificate chain for attestation>]
+  "kid": 0,
+  "key_attestation": [<key attestation in JWT format>]
```

Comment:
This shouldn't be an array
```suggestion
  "key_attestation": <key attestation in JWT format>
```

Reply:
if people want a certificate chain, there is the option to include x5c

Reply:
so that's an argument not to have an array, no, Paul?

Reply:
I take this as an agreement from you also @Sakurann :)